src/Version.cpp

```diff
@@ -0,0 +1,32 @@
+#include <string>
+#include <Gosu/Version.hpp>
+#include <Gosu/Platform.hpp>
+
+namespace Gosu {
+  const std::string VERSION = std::to_string(GOSU_MAJOR_VERSION) + '.' + std::to_string(GOSU_MAJOR_VERSION) + '.' + std::to_string(GOSU_POINT_VERSION);
+
+#ifdef GOSU_IS_WIN
+  const std::string LICENSES =
+      "This software uses the following third-party libraries:\n"
+      "\n"
+      "Gosu, https://www.libgosu.org, MIT License, http://opensource.org/licenses/MIT\n"
+      "SDL 2, http://www.libsdl.org, MIT License, http://opensource.org/licenses/MIT\n"
+      "libsndfile, http://www.mega-nerd.com/libsndfile, GNU LGPL 3, http://www.gnu.org/copyleft/lesser.html\n"
+      "OpenAL Soft, http://kcat.strangesoft.net/openal.html, GNU LGPL 2, http://www.gnu.org/licenses/old-licenses/lgpl-2.0.html\n";
```

Comment:
It's shorter (more "DRY") to only have one string and `#ifdef` specific lines of it